google_guest_agent/non_windows_accounts.go

```diff
@@ -197,6 +182,26 @@ func (a *accountsMgr) set() error {
 	return nil
 }
 
+func getNonExpiredKeys(mdkeys []string) map[string][]string {
```

Comment:
i feel like we do almost this exact same thing in the guest agent code. could that code be reused, instead of reinvented?